ir/src/main/scala/org/scalajs/core/ir/Tags.scala

```diff
@@ -67,7 +67,7 @@ private[ir] object Tags {
   final val TagJSEnvInfo = TagJSObjectConstr + 1
 
   final val TagUndefined = TagJSEnvInfo + 1
-  final val TagUndefinedParam = TagUndefined + 1
+  final val TagUndefinedParam = TagUndefined + 1 // TODO Move this
```

Comment:
Yes. It can be fixed only when we can break binary compatibility, like the other TODOs in this file.